spec/controllers/admin/contacts_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe Admin::ContactsController, :type => :controller do
+
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
+  }
+
+  let(:invalid_attributes) {
+    skip("Add a hash of attributes invalid for your model")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.